libs/mark-flow-ui/src/hooks/use_session_settings_manager.test.tsx

```diff
@@ -11,30 +12,40 @@ import {
   mockCardlessVoterUser,
   mockElectionManagerUser,
   mockSessionExpiresAt,
-  mockUseAudioControls,
-  mockOf,
   TestLanguageCode,
 } from '@votingworks/test-utils';
 import { AudioControls, InsertedSmartCardAuth } from '@votingworks/types';
 import { act, renderHook } from '../../test/react_testing_library';
 import { useSessionSettingsManager } from './use_session_settings_manager';
 
-const mockAudioControls = mockUseAudioControls();
```

Comment:
This is unfortunate. Maybe we should investigate the approach I first tried where I passed the `fn` implementation to use, i.e. `mockUseAudioControls(vi.fn)`.

Reply:
Rebased and updated to use `mockUseAudioControls` from test-utils